catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/Context.java

```diff
@@ -30,6 +31,8 @@
 
     private Optional<Metacard> parentMetacard = Optional.empty();
 
+    private final AtomicBoolean isParentDirty = new AtomicBoolean(false);
```

Comment:
It is a fairly widely used term. https://en.wikipedia.org/wiki/Cache_(computing)#Dirty